Header header
string vehicle_id
geometry_msgs/Pose pose
geometry_msgs/Vector3 accel
geometry_msgs/Vector3 velocity
do8autonomouscar/obstacles[] obstacle_info
do8autonomouscar/lane leftLaneMarkers
do8autonomouscar/lane rightLaneMarkers
